<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <style type="text/css">
  /*
   * Copyright 2013 Christophe-Marie Duquesne <chmd@chmd.fr>
   *
   * CSS for making a resume with pandoc. Inspired by moderncv.
   *
   * This CSS document is delivered to you under the CC BY-SA 3.0 License.
   * https://creativecommons.org/licenses/by-sa/3.0/deed.en_US
   */
  
  /* Whole document */
  body {
      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      width: 800px;
      margin: auto;
      background: #FFFFFF;
      padding: 10px 10px 10px 10px;
  }
  
  /* Title of the resume */
  h1 {
      font-size: 55px;
      color: #757575;
      text-align:center;
      margin-bottom:15px;
  }
  h1:hover {
      background-color: #757575;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Titles of categories */
  h2 {
      /* This is called "sectioncolor" in the ConTeXt stylesheet. */
      color: #397249;
  }
  /* There is a bar just before each category */
  h2:before {
      content: "";
      display: inline-block;
      margin-right:1%;
      width: 16%;
      height: 10px;
      /* This is called "rulecolor" in the ConTeXt stylesheet. */
      background-color: #9CB770;
  }
  h2:hover {
      background-color: #397249;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Definitions */
  dt {
      float: left;
      clear: left;
      width: 17%;
      /*font-weight: bold;*/
  }
  dd {
      margin-left: 17%;
  }
  p {
      margin-top:0;
      margin-bottom:7px;
  }
  
  /* Blockquotes */
  blockquote {
      text-align: center
  }
  
  /* Links */
  a {
      text-decoration: none;
      color: #397249;
  }
  a:hover, a:active {
      background-color: #397249;
      color: #FFFFFF;
      text-decoration: none;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Horizontal separators */
  hr {
      color: #A6A6A6;
  }
  
  table {
      width: 100%;
  }
  </style>
</head>
<body>
<h1 id="scott-allen-funkhouser">Scott Allen Funkhouser</h1>
<hr />
<blockquote>
<p>Updated January 2017</p>
</blockquote>
<hr />
<h2 id="education">Education</h2>
<dl>
<dt>2013-2019</dt>
<dd><strong>PhD, Genetics and Quantitative Biology</strong>; Michigan State University
</dd>
<dt>2007-2011</dt>
<dd><strong>BSc, Biochemistry</strong>; University of Washington
</dd>
<dt>2005-2007</dt>
<dd><strong>Valedictorian</strong>; Graham-Kapowsin HS
</dd>
</dl>
<h2 id="research-positions">Research Positions</h2>
<dl>
<dt>2014 - Present</dt>
<dd><p><strong>Graduate Research Assistant</strong>; Mentor: Catherine Ernst PhD; Michigan State University</p>
<p><em>Investigating the role of A-to-I RNA editing in mammalian genomes and developing methods for the genomic prediction of complex phenotypes.</em></p>
</dd>
<dt>2012 - 2013</dt>
<dd><p><strong>Research Scientist I</strong>; Mentor: Mary Philip MD PhD; University of Washington</p>
<p><em>Investigated the role of a heme exporter, FLVCR, in the development of T lymphocytes.</em></p>
</dd>
<dt>2010 - 2011</dt>
<dd><p><strong>Undergraduate Research Assistant</strong>; Mentor: Chris Hague PhD; University of Washington</p>
<p><em>Studied novel binding partners to pertinent G-Protein coupled receptors.</em></p>
</dd>
</dl>
<h2 id="awards-grants-scholarships">Awards, Grants, Scholarships</h2>
<p><strong>USDA Pre-doctoral Fellowship</strong></p>
<p><em>2017-2019</em> • <em>$95,000</em></p>
<p><strong>ISAG Travel Award</strong></p>
<p><em>$2,000 to attend the 2016 ISAG Conference in Salt Lake City, UT</em></p>
<p><strong>NSIF Graduate Student Travel Award</strong></p>
<p><em>$500 to attend 2016 NSIF meeting in Raleigh, NC</em></p>
<p><strong>Fred and Lucille Stamper Academic College Scholarship</strong></p>
<p><em>2007 - 2011</em> • <em>$16,256</em></p>
<h2 id="memberships">Memberships</h2>
<p><strong>ASAS PhD Student Member</strong></p>
<p><em>2015 – Present</em></p>
<p><strong>Genetics Student Organization Officer</strong></p>
<ul>
<li>2016-Present: <em>President</em></li>
<li>2014-2015: <em>Second-year Representative</em></li>
<li>2015-2016: <em>Third-year Representative and Social Chair</em></li>
</ul>
<h2 id="selected-graduate-coursework">Selected Graduate Coursework</h2>
<ul>
<li>Molecular Biology (BMB 801)</li>
<li>Eukaryotic Molecular Genetics (MMG 835)</li>
<li>R Programming for Quantitative Genetics (ANS 824)</li>
<li>Advanced Statistics for Biologists (STT 814)</li>
<li>Intro to Statistical Genetics (STT 855)</li>
<li>Bayesian Inference using MCMC (FW 849)</li>
</ul>
<h2 id="skills">Skills</h2>
<dl>
<dt><strong>Programming</strong></dt>
<dd><p>Listed in order from most skilled to least skilled</p>
<ul>
<li>The R language</li>
<li>C++</li>
<li>Bash</li>
<li>Fortran</li>
<li>Python</li>
<li>Java</li>
</ul>
<p>Familiar with others for web development including:</p>
<ul>
<li>Markdown</li>
<li>HTML</li>
<li>CSS</li>
<li>Javascript</li>
</ul>
</dd>
<dt><strong>Bioinformatics</strong></dt>
<dd><p>Experience using various tools, including:</p>
<ul>
<li>Bowtie</li>
<li>Tophat</li>
<li>Picard</li>
<li>SAMTools</li>
<li>bcftools</li>
<li>Variant Effect Predictor</li>
<li>UCSC Genome Browser</li>
</ul>
</dd>
<dt><strong>Software development</strong></dt>
<dd><ul>
<li>Git</li>
<li>Github</li>
<li>R Studio</li>
<li>GNU Make</li>
</ul>
<p>R development packages:</p>
<ul>
<li>devtools</li>
<li>roxygen</li>
<li>testthat</li>
</ul>
</dd>
<dt><strong>Literate programming</strong></dt>
<dd><ul>
<li>knitr</li>
<li>Rmarkdown</li>
<li>Shiny</li>
</ul>
</dd>
<dt><strong>Molecular Biology</strong></dt>
<dd><ul>
<li>Flow cytometry</li>
<li>immunostaining</li>
<li>qPCR</li>
<li>PCR</li>
<li>molecular cloning and techniques therein</li>
<li>cell culture</li>
</ul>
</dd>
</dl>
<h2 id="publications">Publications</h2>
<ol style="list-style-type: decimal">
<li><strong>Funkhouser SA</strong>, Bates RO, Ernst CW, Newcom D, Steibel JP. Estimation of genome-wide and locus-specific breed composition in pigs. TAS. 2017; doi: 10.2527/tas2016.0003</li>
<li>Zhao D, Hamilton JP, Hardigan M, Yin D, He T, Vaillancourt B, Reynoso M, <strong>Funkhouser SA</strong> et al. Analysis of ribosome-associated mRNAs in rice reveals the importance of transcript size and GC content in translation. G3 (Bethesda). 2016;7: 203–219. doi:10.1534/g3.116.036020</li>
<li>Philip M, <strong>Funkhouser SA</strong>, Chiu EY, Phelps SR, Delrow JJ, Cox J, et al. Heme Exporter FLVCR Is Required for T Cell Development and Peripheral Survival. J Immunol. 2015; doi:10.4049/jimmunol.1402172</li>
</ol>
<h2 id="pending-publications">Pending Publications</h2>
<ol style="list-style-type: decimal">
<li><strong>Funkhouser SA</strong>, Steibel JP, Bates RO, Raney NE, Schenk D, Ernst CW. Evidence for Transcriptome-wide RNA Editing Among Sus scrofa PRE-1 SINE Elements. bioRxiv. 2017; Available: <a href="http://biorxiv.org/content/early/2017/01/05/096321.abstract">here</a></li>
<li>Reeb PD, <strong>Funkhouser SA</strong>, Ernst CW, Steibel JP. Assessing genotype call accuracy from RNA sequencing data.</li>
</ol>
<h2 id="conference-presentations-and-posters">Conference Presentations and Posters</h2>
<ul>
<li><strong>Funkhosuer SA</strong>, Bates RO, CW Ernst, Newcom D, Steibel JP. (2016) Estimation of the composition of four U.S. swine breeds using genomic data. ePoster presented at the 2016 ASDA/AAS Joint Annual Meeting.</li>
<li><strong>Funkhouser SA</strong>, Steibel JP, Bates RO, Raney NE, Ernst CW. (2016) RNA editing in swine is associated with PRE-1 retrotransposons. ePoster presented at the 2016 International Society for Animal Genetics Conference.</li>
<li><strong>Funkhouser SA</strong>, Steibel JP, Bates RO, Raney NE, Ernst CW. (2015) Evidence of RNA editing in pig longissimus dorsi muscle. Oral presentation given at the American Dairy Science / American Society of Animal Science Midwest Conference. <!-- * Funkhouser SA, Steibel JP, Newcom D, Ernst CW.
Evaluation of four US pig breeds using the Affymetrix Axiom PigHD array. PAG
Swine Workshop January 2017.
* C.W. Ernst, S. A. Funkhouser, J.P. Steibel, R.O. Bates, N.E. Raney. (2015)
Evidence of RNA editing in pig longissimus dorsi muscle.
Oral presentation given at the Plant and Animal Genome XXIII Conference.
* M. Philip, S.A. Funkhouser, J.J. Delrow, E.Y. Chiu, and J.L. Abkowitz. (2012)
FLVCR, a Heme Exporter, Is Required for Peripheral T Cell Survival.
Poster presented at the 54th Annual American Society of Hematology Meeting and
Exposition.
* M. Philip, S.A. Funkhouser, J.J. Delrow, and J.L. Abkowitz. (2012) FLVCR, a Heme
Export Protein, is Required for T Cell Development and Survival. Poster
presented at the Keystone Symposium on Molecular and Cellular Biology --></li>
</ul>
<hr />
<blockquote>
<p><a href="mailto:sfunkhouser@gmail.com">sfunkhouser@gmail.com</a> • (253)-720-2797 • 474 S. Shaw Lane, Room 1205 - East Lansing, Michigan, USA</p>
</blockquote>
</body>
</html>
